core/src/main/java/io/aiven/klaw/service/MailUtils.java

```diff
@@ -236,6 +236,31 @@ void sendMail(
         subject = "New Connector Claim Request";
         formattedStr = "New Claim on Connector " + topicName;
       }
+      case RESET_CONSUMER_OFFSET_REQUESTED -> {
```

Comment:
I think these are good for now but we should ask Harishini for Copy when we get a chance!